themes/src/main/resources/theme/base/admin/resources/partials/realm-tokens.html

```diff
@@ -152,4 +152,4 @@
 
 </div>
 
-<kc-menu></kc-menu>
```

Comment:
Funny, this should not be here - seems I got some garbage in my last rebase . I'll fix it.